src/appmixer/hubspot/routes.js

```diff
@@ -116,35 +116,48 @@ module.exports = async (context) => {
             auth: false,
             handler: async (req) => {
 
+                await context.log('info', 'hubspot-plugin-route-webhook-hit', { payload: req.payload });
```

Comment:
Be careful with logs. For development this is fine. However, logging every incoming payload in production can cause issues. If the payload is big and there are lots of incoming requests, that could take lots of disk space.

Reply:
I put potentially large logs into `trace` level. Also changed the messages (2nd argument) a bit to make it more searchable.